randovania/games/dread/json_data/Elun.txt

```diff
@@ -139,7 +139,7 @@ Extra - asset_id: collision_camera_003
   * Extra - right_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:door_shield_plasma_004
   * Extra - right_shield_def: actordef:actors/props/door_shield_plasma/charclasses/door_shield_plasma.bmsad
   > Door to Chozo Soldier Arena
-      Walljump (Beginner) or Use Spin Boost
+      Use Spin Boost
```

Comment:
If you're gonna go through with changing these, anything where a beginner walljump alone is enough, the whole thing can just be made trivial